### ISSUE TITLE
Display current testscript used

### ISSUE THREAD

**Issue by tyge68** [locustio/locust#1832]
### Is your feature request related to a problem? Please describe.
Currently when we start locust ui , there are no hints on the page to find out what will be executed exactly.
### Describe the solution you'd like
It would be nice to at least display the current testscript filename that was used to start the command a.k.a -f option value.
If not at least the list of tasks that are going to be executed would be also nice to have.
### Describe alternatives you've considered
I have no idea of any possible alternative.

### Additional context
Distributed installation with many worker machines and a dedicated master running with UI.

**Comment by cyberw** [comment]
Good idea! Shouldnt be too difficult to do either. Maybe have a go att doing it yourself?

**Comment by tyge68** [comment]
@cyberw not sure how that could be done by myself unfortunately. I guess more experienced person on this project could add this quicker.

**Comment by cyberw** [comment]
I’m sure it would be quicker, but it is unlikely to happen. Particularly the UI is ”you want it, you build it”, as I dont use it myself :)

**Comment by tyge68** [comment]
is there information how to build locust locally ? or can we simply do pip install . maybe ? not sure, I am not really expert in this area :(

**Comment by cyberw** [comment]
Clone the repo from github, cd into the folder and run 
pip3 install -e .

**Comment by tyge68** [comment]
@cyberw  thanks a lot, I managed to do something, you can check my draft PR on #1833
